src/components/Search/AdvancedSearch.tsx

```diff
@@ -0,0 +1,59 @@
+import { useState } from "react";
+import { Stack } from "@mui/material";
+import {
+  CheckInInput,
+  CheckOutInput,
+  GuestNumberInput,
+  AccommodationTypeInput,
+} from "../InputFields";
+import { SearchButton } from "../Buttons/SearchButton";
+
+export const AdvancedSearch = (): JSX.Element => {
+  const [checkIn, setCheckIn] = useState("");
+  const [checkOut, setCheckOut] = useState("");
+  const [guestNumber, setGuestNumber] = useState("");
+  const [accommodationType, setAccommodationType] = useState("");
+
+  const getData = () => {
+    console.log("Submitaj podatke");
+    const accommodationSearchSubmit = {
+      checkIn: checkIn,
+      checkOut: checkOut,
+      guestNumber: guestNumber,
+      accommodationType: accommodationType,
+    };
+    console.log(accommodationSearchSubmit);
+  };
+
+  const getCheckInData = (props: string) => {
+    setCheckIn(props);
+  };
+
+  const getCheckOutData = (props: string) => {
+    setCheckOut(props);
+  };
+
+  const getGuestNumberData = (guestNumberInput: string) => {
+    setGuestNumber(guestNumberInput);
+  };
+
+  const getAccommodationTypeData = (accommodationTypeInput: string) => {
+    setAccommodationType(accommodationTypeInput);
+  };
+  return (
+    <Stack
+      direction="row"
+      spacing={2}
+      sx={{
+        display: "flex",
+        alignItems: "center",
+      }}
+    >
+      <CheckInInput getCheckIn={getCheckInData} />
```

Comment:
Surround with `<Form onSubmit={...}>`

Reply:
Also add in the function `e.preventDefault();` so the form doesn't redirect by default.